.github/actions/build-docs/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Documentation
+description: 'Build Documentation.'
+inputs:
+  workspace_path:
+    description: 'Path to the workspace.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Setup dotnet build tools
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 6.0
+    - name: Install docfx
+      shell: bash
+      run:  dotnet tool update -g docfx
+
+    # Note: in the docfx.json file, the 'Configuration' property is set to Debug so that we don't require
+    # signing to happen just to build docs.
+    - name: Build docs
+      shell: bash
+      run: |
+        pushd ${{ inputs.workspace_path }}
+        docfx metadata
```

Comment:
Have you tried this out locally? I had found that the version of docfx used by releaser was very ancient and required configuration changes to get usable documentation (adding a docfx with the configuration).

Reply:
(Some already are like dotnet-eventsource).

Reply:
Good catch. It looks like we don't even build docs for the repo at the moment in releaser. I did run the commands and brought in the config file but it had some issues so I'm going to remove it and we can add it in later if we want it.

Reply:
Ah, yeah, makes sense we may not even want to bother with API docs for this package.